script/Ciel/Day 7/07_01_CIEL07_3.txt

```diff
@@ -416,6 +416,7 @@ Well, so much for whatever errand she was supposed to be running.
 -- Page 382, Offset 15734.
 -- お姫様の気まぐれですっぽかされる立場に同情しないでもないが、それはともかく、
 // double negative preserved, but could be shortened if the need is so felt.
+// why the sudden change to Her Majesty? it's always been princess or dear princess
```

Comment:
In this case it adds a bit of a mocking tone that I think fits well

Reply:
I think dear princess also has that mocking tone but sure a little variety wont hurt